.circleci/config.yml

```diff
@@ -10,6 +10,12 @@ jobs:
             bundler_version:
                 type: string
                 default: 2.2.29
+            postgres_version:
+                type: string
+                default: 9.6.2-alpine
+            solr_version:
+                type: string
+                default: 7-alpine
```

Comment:
I think we want to user Solr 8.x don't we?

(Because that's what I said to use here: https://github.com/curationexperts/tenejo/blob/main/.solr_wrapper.yml#L2)